samples/wifi/provisioning/softap/README.rst

```diff
@@ -0,0 +1,51 @@
+.. _softap_wifi_provision_sample:
+
+Wi-Fi: SoftAP provision
+#######################
+
+.. contents::
+   :local:
+   :depth: 2
+
+This sample demonstrates how to use the :ref:`lib_softap_wifi_provision` library to provision credentials to a Wi-Fi® device (nRF70 Series).
+
+.. |wifi| replace:: Wi-Fi
+
+.. include:: /includes/net_connection_manager.txt
+
+Requirements
+************
+
+The sample supports the following development kits:
+
+.. table-from-sample-yaml::
+
+.. include:: /includes/tfm.txt
+
+Configuration
+*************
+
+|config|
+
+Configuration files
+===================
+
+The sample includes the following pre-configured configuration file for the development kits that are supported:
+
+* :file:`prj.conf` - Configuration file for all build targets.
+
+Building and running
```

Comment:
Does the sample require a corresponding mobile app counterpart? how do we suggest developers testing it? It would be nice to add a couple of sentences about this here.

Reply:
I plan to extend the docs after the initial commit has beeen made. Good idea to mention this.  We are not dependent on the mobile app.

Reply:
That's ok